bundles/org.openhab.core.io.rest.core/src/test/java/org/openhab/core/io/rest/core/config/EnrichedConfigDescriptionDTOMapperTest.java

```diff
@@ -95,9 +95,9 @@ public void testThatDefaultValuesAreAList() {
 
     @Test
     public void testThatDefaultValuesDontSplitEscapedCommas() {
-        final String CONFIG_PARAMETER_DEFAULT_VALUE = "Me\\, myself\\, and I,You \\\\,";
+        final String config_parameter_default_value = "Me\\, myself\\, and I,You \\\\,";
```

Comment:
Underscores are only used with upper case constants:

```suggestion
        final String configParameterDefaultValue = "Me\\, myself\\, and I,You \\\\,";
```